src/test/java/core/basesyntax/db/strategy/handler/impl/PurchaseOperationHandlerTest.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax.db.strategy.handler.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.db.model.FruitTransaction;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class PurchaseOperationHandlerTest {
+    private static PurchaseOperationHandler purchaseOperationHandler;
```

Comment:
```suggestion
    private static OperationHandler purchaseOperationHandler;
```
the same for other implementations test classes